src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
-// write your code here
+document.querySelector('.logo').addEventListener('click', () => {
+  callPromise1();
+  callPromise2();
+});
+
+function callPromise1() {
+  const promise1 = new Promise((resolve, reject) => {
+    resolve(createMsg('', 'Promise was resolved!'));
```

Comment:
```suggestion
    resolve(createMsg('Promise was resolved!'));
```
better then to change the order of parameters

Reply:
Suggested change are implemented